components/typography/TextLink.js

```diff
@@ -8,12 +8,14 @@ export default function TextLink({
 
     const classNames = `text-secondary-500 underline hover:text-gray-800 font-medium transition ease-in-out duration-300 ${className || ''}`
 
-    return(
+    return (
         <>
-        {' '}
-        <a href={href} target={target} rel="noreferrer noopener" className={classNames}>
-            {children}
-        </a>
+            {' '}
+            <Link href={href}>
```

Comment:
Same reason given in above comment.

Reply:
I think that it's better to have only a tag for external ones. Please check first if href starts by http(s) and only if yes, apply the Link component.